control-plane/msp-operator/src/main.rs

```diff
@@ -1021,7 +1021,20 @@ async fn main() -> anyhow::Result<()> {
         return write_msp_crd(matches.value_of("write_crd"));
     }
 
-    pool_controller(matches).await?;
+    let mut signal_term = tokio::signal::unix::signal(tokio::signal::unix::SignalKind::terminate())
+        .expect("Failed to register handler for SIGTERM");
+    let mut signal_int = tokio::signal::unix::signal(tokio::signal::unix::SignalKind::interrupt())
+        .expect("Failed to register handler for SIGINT");
+
+    tokio::select! {
+        _evt = signal_term.recv() => {
+            }
+        _evt = signal_int.recv() => {
+            }
+        _ps = pool_controller(matches) => {
+            _ps?
```

Comment:
`_` is only used if you don't use the variable

Reply:
fixed  https://github.com/mayadata-io/mayastor-control-plane/pull/158/commits/2c281922104a3665655f0dddce06f56254e18e01